### PR TITLE
Documentation for openshift ex generate command

### PR DESCRIPTION
**Pull request by csrwng**
Adds documentation for the openshift experimental generate command

### CODE REVIEW

**Comment by bparees** [comment]
@csrwng is this ready for the docs team to take a look at/merge?

**Comment by csrwng** [comment]
@bparees yes

**Comment by bparees** [comment]
@CowboysFan @adellape can one of you take a look?

**Comment by adellape** [comment]
@csrwng LGTM; rebase and I'll merge?

File: using_openshift/generate_cmd.adoc [hunk 1]
```diff
@@ -0,0 +1,107 @@
+= Command Line Interface
```
Comment (adellape) [inline using_openshift/generate_cmd.adoc]: Just noticed this, which I assume was just a copy/paste remnant. Even though the `Name:` value in `_build_cfg.yml` seems to determine the topic title that is actually rendered in builds, this line might as well be updated to match (e.g., "Config Generation").

**Comment by CowboysFan** [comment]
@csrwng @adellape Where are we with this? Is this ready to be merged?

**Comment by csrwng** [comment]
@CowboysFan should be ready now. I've rebased and made the change requested by @adellape

**Pull request merged**